docs/api/sql/Raster-aggregate-function.md

```diff
@@ -15,28 +15,37 @@ Since: `v1.5.1`
 
 SQL Example
 
-Contents of `raster_table`.
+```
+val windowSpec = Window.orderBy("timestamp")
+val indexedRasters = df.withColumn("index", row_number().over(windowSpec))
+
+indexedRasters.show()
+```
 
 ```
-+------------------------------+-----+
-|                        raster|index|
-+------------------------------+-----+
-|GridCoverage2D["geotiff_cov...|    1|
-|GridCoverage2D["geotiff_cov...|    2|
-|GridCoverage2D["geotiff_cov...|    3|
-|GridCoverage2D["geotiff_cov...|    4|
-|GridCoverage2D["geotiff_cov...|    5|
-+------------------------------+-----+
++-----+-------------------+------------------------------+
+|index|          timestamp|                        raster|
++-----+-------------------+------------------------------+
+|    1|2021-01-01T00:00:00|GridCoverage2D["geotiff_cov...|
+|    2|2021-01-02T00:00:00|GridCoverage2D["geotiff_cov...|
+|    3|2021-01-03T00:00:00|GridCoverage2D["geotiff_cov...|
+|    4|2021-01-04T00:00:00|GridCoverage2D["geotiff_cov...|
+|    5|2021-01-05T00:00:00|GridCoverage2D["geotiff_cov...|
++-----+-------------------+------------------------------+
 ```
 
 ```
-SELECT RS_Union_Aggr(raster, index) FROM raster_table
+SELECT RS_Union_Aggr(raster, index) AS raster, RS_NumBands(raster) AS Num_Bands FROM raster_table
 ```
 
 Output:
 
-This output raster contains the first band of each raster in the `raster_table` at specified index.
+This output raster contains all bands of each raster in the `raster_table`.
```

Comment:
Can we show a groupby example as well? Grouping by time and geography, similar to what we are doing with the segmentation dataset prep?